### DISCUSSION TITLE
How to reinstall eWeLinkControlPanel?

### DISCUSSION THREAD

**Discussion by christescu** [seaky/nspanel_pro_tools_apk#54] [Q&A]
I have removed bloated applications, including eWeLinkControlPanel.
Is any way to reinstall it?
I have factory version 1.4, updated to 1.5 with nspanel_pro_roottool.

**Comment by svanrossem** [comment]
```
px30_evb:/ $ pm list packages | grep eWeLinkControlPanel                                                 
package:com.eWeLinkControlPanel
px30_evb:/ $ pm path com.eWeLinkControlPanel                                                             
package:/system/app/kz_app/kz_app.apk
```

You could try installing `/system/app/kz_app/kz_app.apk`.

**Reply by christescu** [reply]
For me looks different:
```
px30_evb:/ $ pm list packages | grep eWeLinkControlPanel
1|px30_evb:/ $ pm path com.eWeLinkControlPanel
1|px30_evb:/ $
```

I also pull `/system/app/kz_app/kz_app.apk` and have a look inside, nothing related to eWeLinkControlPanel.
